go.mod

```diff
@@ -11,20 +11,23 @@ replace (
 )
 
 require (
+	github.com/Azure/go-autorest/autorest/azure/auth v0.4.2 // indirect
```

Comment:
Why are the dependencies for `github.com/Azure/go-autorest/autorest/azure/auth`, `github.com/masterzen/winrm` and `github.com/pkg/sftp` added?

Reply:
Thanks for catching this. I have fixed it.